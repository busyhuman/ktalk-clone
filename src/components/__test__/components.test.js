/* eslint-disable no-undef */
import Vue from 'vue'
import Alarm from '../modal/AlarmModalComponent.vue'
import ModalBack from '../modal/ModalBackComponent.vue'
import Chat from '../ChatComponent.vue'
import Nav from '../NavComponent.vue'
import NormalList from '../NormalListComponent.vue'
import Profile from '../ProfileComponent.vue'

jest.mock('../ChatComponent.vue')
jest.mock('../NavComponent.vue')
jest.mock('../NormalListComponent.vue')
jest.mock('../ProfileComponent.vue')

describe('Component', () => {
  test('[Snapshot Testing] Alarm', () => {
    const cmp = new Vue(Alarm).$mount()
    expect(cmp.$el).toMatchSnapshot()
  })

  test('[Snapshot Testing] ModalBack', () => {
    const cmp = new Vue(ModalBack).$mount()
    expect(cmp.$el).toMatchSnapshot()
  })

  test('[Snapshot Testing] Chat', () => {
    const cmp = new Vue(Chat).$mount()
    expect(cmp.$el).toMatchSnapshot()
  })

  test('[Snapshot Testing] Nav', () => {
    const cmp = new Vue(Nav).$mount()
    expect(cmp.$el).toMatchSnapshot()
  })

  test('[Snapshot Testing] NormalList', () => {
    const cmp = new Vue(NormalList).$mount()
    expect(cmp.$el).toMatchSnapshot()
  })

  test('[Snapshot Testing] Profile', () => {
    const cmp = new Vue(Profile).$mount()
    expect(cmp.$el).toMatchSnapshot()
  })
})

