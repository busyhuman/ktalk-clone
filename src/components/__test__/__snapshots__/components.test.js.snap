// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component [Snapshot Testing] Alarm 1`] = `undefined`;

exports[`Component [Snapshot Testing] Chat 1`] = `undefined`;

exports[`Component [Snapshot Testing] ModalBack 1`] = `undefined`;

exports[`Component [Snapshot Testing] Nav 1`] = `undefined`;

exports[`Component [Snapshot Testing] NormalList 1`] = `undefined`;

exports[`Component [Snapshot Testing] Profile 1`] = `undefined`;
